modules/ingester/instance.go

```diff
@@ -111,7 +118,7 @@ func (i *instance) CutCompleteTraces(cutoff time.Duration, immediate bool) error
 			if err != nil {
 				return err
 			}
-
+			i.bytesProcessedTotal.Add(float64(len(out)))
```

Comment:
Should this be after the err check?  Depends on how we define the metric, as attempted vs succeeded.

Reply:
I went with the attempted approach. But probably succeded makes more sense here :thinking: 

btw, I agree with the metric name. Love your proposal! I'll change it.

Reply:
So I just want to make a note about this spot:

- This is occurring after a trace is considered complete, so it will be delayed a bit while the trace is aggregating spans.  - On rollout we'd expect to see this metric to spike while ingesters are flushing their in memory traces to disk
- This metric is post RF.  So it will be ~1x, ~2x or ~3x the number of pushed bytes depending on that setting.

Reply:
Moved :)